.changeset/fast-berries-sneeze.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+Merge b46be6c49c490b2116ebed84ec254e0acafb8199 into bbd31a192e4de862c653fb1d30e813773716df43
```

Comment:
Looks like we are [in the same boat](https://github.com/FuelLabs/fuels-ts/actions/runs/8879891098/job/24378703778?pr=2191#step:5:20) @nedsalk 😓